src/mas/devops/templates/pipelinerun-install.yml.j2

```diff
@@ -308,6 +312,19 @@ spec:
     - name: sls_action
       value: "{{ sls_action }}"
 {%- endif %}
+{%- if aiservice_sls_url is defined and aiservice_sls_url != "" %}
```

Comment:
We should not need these in the pipeline params ... sls info will come from the SLS we installed as part of the pipeline.  The reference topology supported by the CLI functions does not support SLS external to the cluster because it's considered an anti-pattern than makes day2 operations problematic.